observers/qos_correction.hpp

```diff
@@ -0,0 +1,132 @@
+#ifndef SERENITY_QOS_CORRECTION_HPP
+#define SERENITY_QOS_CORRECTION_HPP
+
+#include <list>
+#include <vector>
+
+#include "glog/logging.h"
+
+#include "mesos/mesos.hpp"
+#include "mesos/resources.hpp"
+
+#include "mesos/slave/oversubscription.hpp"
+
+#include "messages/serenity.hpp"
+
+#include "serenity/serenity.hpp"
+
+namespace mesos {
+namespace serenity {
+
+inline std::list<ResourceUsage_Executor> filterPrExecutors(
```

Comment:
Mind documenting this helper? I have a hard time understanding what 'filterPerExecutors' does